### DISCUSSION TITLE
10G KR phy driver issue

### DISCUSSION THREAD

**Discussion by duaneellissd** [Mi-V-Soft-RISC-V/miv-rv32-documentation#16] [Bare Metal Embedded Software] [answered]
Where is this include file located?

https://github.com/Mi-V-Soft-RISC-V/platform/blob/main/drivers/fpga_ip/Core10gBaseKR_PHY/phy_types.h#L38

**Comment by nitindeshpande** [comment, marked as answer]
Please refer https://github.com/Mi-V-Soft-RISC-V/platform/blob/b2a0ae7b25608477e27e1fe078da180dd5d31c7a/drivers/fpga_ip/Core10gBaseKR_PHY/phy.h#L56
